1_uprava.js

```diff
@@ -0,0 +1,8 @@
+const countVowels = (text) => {
+    return text.match(/[aeiou]/gs).length;
+}
+
+const input = "testing vowel text";
+const finalCount = countVowels(input);
+
+console.log(`Input text: \n"${input}"\nhas ${finalCount} vowels`);
```

Comment:
Opat iba ES6 improvement. Namiesto chainovania textu pluskami mozes pouzit `literal strings`.
Ide o to ze ak obalis text v ` uvodzovkach tak tam mozes priamo vkladat premenne pomocou `${premenna}`.
Ide zase o zjednodusenie zapisu ako aj citatelnosti